Gems/SimulationInterfaces/Code/Source/Clients/SimulationEntitiesManager.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzCore/Component/TickBus.h>
+#include <AzCore/Script/ScriptTimePoint.h>
+#include <AzFramework/Entity/EntityContextBus.h>
+#include <AzFramework/Physics/PhysicsScene.h>
+#include <AzFramework/Spawnable/SpawnableEntitiesInterface.h>
+#include <SimulationInterfaces/SimulationEntityManagerRequestBus.h>
+
+namespace SimulationInterfaces
+{
+    class SimulationEntitiesManager
+        : public AZ::Component
+        , protected SimulationEntityManagerRequestBus::Handler
+    {
+    public:
+        AZ_COMPONENT_DECL(SimulationEntitiesManager);
+
+        static void Reflect(AZ::ReflectContext* context);
+
+        static void GetProvidedServices(AZ::ComponentDescriptor::DependencyArrayType& provided);
+        static void GetIncompatibleServices(AZ::ComponentDescriptor::DependencyArrayType& incompatible);
+        static void GetRequiredServices(AZ::ComponentDescriptor::DependencyArrayType& required);
+        static void GetDependentServices(AZ::ComponentDescriptor::DependencyArrayType& dependent);
+
+        SimulationEntitiesManager();
+        ~SimulationEntitiesManager();
+
+    protected:
```

Comment:
The manager has no subclasses and by other parts of design (such as non-virtual destructor) indicate is not intended for subclass, there should be no "protected:" section at all, all this can be private.